uniffi_bindgen/src/bindings/kotlin/gen_kotlin/mod.rs

```diff
@@ -343,6 +343,27 @@ pub mod filters {
         Ok(oracle().ffi_type_label(type_))
     }
 
+    /// Get the type that a type is lowered into.  This is subtly different than `type_ffi`, see
+    /// #1106 for details
```

Comment:
It's the same spirit as in `swift/gen_swift/mod.rs`. Do you want me to update this Swift impl' too?

Reply:
For Swift, I added the extra function because types were named differently in the `.h` and `.swift` files.  I don't really like the system, but it works okay.

For Kotlin, we only generate one file, so why do we need two different functions?